fix/upp/postxconfig-NT.txt

```diff
@@ -984,6 +984,48 @@ surface
 ?
 ?
 ?
+218
+VGTYP_ON_SURFACE
```

Comment:
Why we add a new variable? Do we need the latest UPP to handle this? Thanks!

Reply:
@guoqing-noaa it appears the rrfs-workflow mpas_hrrr_postcntrl.xml and postxconfig-NT.txt are not consistent, because I did not add or change anything related to VGTYP in mpas_hrrr_postcntrl.xml.  I will try to make them consistent...

Reply:
@guoqing-noaa OK I think the mpas_hrrr_postcntrl.xml and postxconfig-NT.txt are consistent now.  Let me just test it with a UPP run.

Reply:
@guoqing-noaa do you have test MPAS files?  Looks like my previous UPP tests just had soft links to your MPASSIT output, which is now gone

Reply:
@EricJames-NOAA Here is a UPP case:
```
/scratch1/BMC/wrfruc/gge/nco/stmp/conus12km/1.0.1/rrfs.20240527/00/upp/001
```
Thanks!